ios/Classes/SwiftFlutterUploaderPlugin.swift

```diff
@@ -278,11 +278,12 @@ public class SwiftFlutterUploaderPlugin: NSObject, FlutterPlugin, URLSessionTask
                                                        completion completionHandler:@escaping (URLSessionUploadTask?, FlutterError?) -> Void) {
         let request = NSMutableURLRequest(url: url)
         request.httpMethod = method
-        request.addValue("*/*", forHTTPHeaderField: "Accept")
+        request.setValue("application/json; charset=utf-8", forHTTPHeaderField: "Accept")
```

Comment:
The content type is configurable below but fixed to `application/JSON` here, can this be changed so it’s configurable both times?

Reply:
So you mean to set it to `request.setValue("*/*", forHTTPHeaderField: "Accept")` right ?

Reply:
Yes.. in theory the plugin should not expect/restrict a server response.

Reply:
Yup my bad. Will change it soon.